webplugin/js/app/mck-app.js

```diff
@@ -310,6 +310,17 @@ function ApplozicSidebox() {
             var widgetSettings = data.chatWidget;
             var disableChatWidget = options.disableChatWidget != null ? options.disableChatWidget : widgetSettings.disableChatWidget; // Give priority to appOptions over API data.
             
+            var allowedDomains = widgetSettings.allowedDomains;
+            var hostname = parent.window.location.hostname;
+
+            const isSubDomain = (domain) => hostname.endsWith(domain);
```

Comment:
@shubhamtibra endsWith property is not supported in IE.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith